packages/web-react/src/types/checkboxField.ts

```diff
@@ -1,20 +1,31 @@
-import { ChildrenProps, StyleProps, InputProps, ItemProps, TransferProps, HelperTextProps } from './shared';
+import {
+  ChildrenProps,
+  HelperTextProps,
+  InputBaseProps,
+  ItemProps,
+  SpiritInputElementProps,
+  Validation,
+} from './shared';
 import { LabelProps } from './label';
 import { MessageProps } from './message';
 
+export type CheckboxElementBaseProps = Omit<SpiritInputElementProps, 'type' | 'required'>;
```

Comment:
Instead of omitting I would suggest creating `InputTypeProp` and `RequiredInput` and extending these types.

Reply:
I'm sorry, but I don't understand your comment at all.

I have to throw out these types in the final, because we don't need them or we work with them differently (in the case of type). So it applies to every Field.

How should I create `type InputTypeProp` and redefine it? And then I add it to what?

Reply:
Ok, got it

Reply:
done